python/ray/serve/common.py

```diff
@@ -49,24 +48,28 @@ class ReplicaName:
     replica_suffix: str
     replica_tag: ReplicaTag = ""
     delimiter: str = "#"
+    prefix: str = "SERVE_REPLICA::"
```

Comment:
what if we keep the prefix in replica name for serve such that its internal name & actor name always have it, rather than replaced ? `SERVE_REPLICA::DeploymentA#zxcvbn` I also don't find where we provided `SERVE_REPLICA::` such that 

```
all_replica_names = [
    actor_name for actor_name in all_current_actor_names
    if ReplicaName.is_replica_name(actor_name)
] 
```

will return true from actor names found in current ray cluster.